app/v2/generators/DepartureMessageGenerator.scala

```diff
@@ -266,4 +270,85 @@ class DepartureMessageGeneratorImpl @Inject()(clock: Clock) extends Generators w
         </Consignment>
       </ncts:CC029C>
     )
+
+  private def generateIE056Message(correlationId: String): XMLMessage =
+    XMLMessage(
+      <ncts:CC056C PhaseID="NCTS5.0" xmlns:ncts="http://ncts.dgtaxud.ec">
+        <messageSender>{alphanumeric(35)}</messageSender>
+        <messageRecipient>{correlationId}</messageRecipient>
+        <preparationDateAndTime>{generateLocalDateTime()}</preparationDateAndTime>
+        <messageIdentification>{alphanumeric(35)}</messageIdentification>
+        <messageType>CC056C</messageType>
+        <correlationIdentifier>{correlationId}</correlationIdentifier>
+        <TransitOperation>
+          <LRN>{Strings.alphanumeric(2, 22)}</LRN>
+          <MRN>{Strings.mrn()}</MRN>
+          <businessRejectionType>{alphanumeric(3)}</businessRejectionType>
+          <rejectionDateAndTime>{generateLocalDateTime()}</rejectionDateAndTime>
+          <rejectionCode>{numeric(2)}</rejectionCode>
```

Comment:
This should be numeric(1, 2)
 <xs:restriction base="xs:token">
      <xs:pattern value="[0-9]{1,2}" />
    </xs:restriction>